src/choropleth.ts

```diff
@@ -666,6 +666,12 @@ function showMapCallout(data, event, d): void {
   // set the callout title text
   callout.select("#map-callout-title").text(d.properties.name);
 
+  if (d.properties.GEOID10) {
```

Comment:
Nit: This makes me a bit itchy because we don't have a consistently applied single representation of what geo levels. 
 
IIRC, this is saying that if we have a `GEOID10` property that means we're looking at ZIP data and there's no further level to drill down into.  Semantically, this would read better if the test said, `if(sub_region_3)` or if we had an actual tree data structure `if(!hasChildren)`.

Is it worth fixing this for real right now?  Probably not.

Reply:
I think we should take care of that as part of the map component refactor.